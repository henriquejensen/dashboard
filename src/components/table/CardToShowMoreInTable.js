import React from 'react';
import { Col } from 'react-bootstrap';

import { NENHUM_REGISTRO } from "../../constants/utils";

const CardToShowMoreInTable = (props) => {
    return (
        <div>
            {props.elements.map((element, index) => {
                return (
                    <Col md={4} key={index}><strong>{element.label}</strong> {": " + (element.value ? element.value : NENHUM_REGISTRO)}</Col>
                )
            })}
            
        </div>
    );
};

export default CardToShowMoreInTable;