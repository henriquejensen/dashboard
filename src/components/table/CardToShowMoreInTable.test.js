import React from 'react';
import renderer from "react-test-renderer";
import CardToShowMoreInTable from "./CardToShowMoreInTable";

describe("Testando o CardToShowMoreInTable", () => {
    it('', () => {
        const tree = renderer.create(<CardToShowMoreInTable elements={[{label:"Label 1", value:5},{label:"Label 2", value:10}]} />);

        expect(tree.toJSON()).toMatchSnapshot();
    })
});