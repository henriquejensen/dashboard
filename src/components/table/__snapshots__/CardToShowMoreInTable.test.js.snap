// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando o CardToShowMoreInTable  1`] = `
<div>
  <div
    className="col-md-3"
  >
    Label 1: 5
  </div>
  <div
    className="col-md-3"
  >
    Label 2: 10
  </div>
</div>
`;
